scripts/serverless.js

```diff
@@ -594,10 +595,9 @@ processSpanPromise = (async () => {
       configuration,
       serviceDir,
       configurationFilename,
-      isConfigurationResolved:
-        commands[0] === 'plugin' || Boolean(variablesMeta && !variablesMeta.size),
```

Comment:
I've removed it `main` branch. Please rebase

Reply:
Rebased